Formula/csound.rb

```diff
@@ -39,6 +40,13 @@ def install
     mkdir "build" do
       system "cmake", "..", *args
       system "make", "install"
+
+      macho = MachO.open("lib_jcsound6.jnilib")
+      macho.change_install_name("@rpath/libcsnd6.6.0.dylib", "#{opt_prefix}/lib/libcsnd6.dylib")
+      macho.change_install_name("@rpath/CsoundLib64.framework/Versions/6.0/CsoundLib64", "#{opt_prefix}/Frameworks/CsoundLib64.framework/CsoundLib64")
```

Comment:
Why is this needed? What's wrong with rpath?

Reply:
The most honest answer I can give, unfortunately, is: I really don’t know. The Mach-O stuff isn’t necessary on my machine. With all the Mach-O stuff deleted, I can run—

```sh
brew install --build-from-source csound
brew test csound
```

—and the [Java bindings test](https://github.com/Homebrew/homebrew-core/pull/40327/files#diff-287bbbfba06a8ff0471bc82fcefc18eaR109) works fine.

The Mach-O stuff [wasn’t part](https://github.com/Homebrew/homebrew-core/pull/40327/commits/69e630a63d31e251013f1bc72daaeb31559b8392#diff-287bbbfba06a8ff0471bc82fcefc18eaR42) of this pull request originally, but the Java bindings test failed on Jenkins. (It seems like these test results should be available as [build #42678](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/42678/), but I get a 404 error. Builds between [#38238](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/38238/) and [#43089](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/43089/) appear to be missing.) If you want, I can remove the Mach-O stuff and see what the error was on Jenkins.

Reply:
Please do.

And old builds get removed so will respond with 404 after a while.

Reply:
After removing the Mach-O stuff, the test fails with:

```
==> java -Djava.library.path='/usr/local/opt/csound/libexec' test
Picked up _JAVA_OPTIONS:  -Duser.home=/Users/brew/Library/Caches/Homebrew/java_cache
_jcsound6 native code library failed to load.
java.lang.UnsatisfiedLinkError: /usr/local/Cellar/csound/6.12.2_3/libexec/lib_jcsound6.jnilib: dlopen(/usr/local/Cellar/csound/6.12.2_3/libexec/lib_jcsound6.jnilib, 1): Library not loaded: @rpath/CsoundLib64.framework/Versions/6.0/CsoundLib64
  Referenced from: /usr/local/Cellar/csound/6.12.2_3/libexec/lib_jcsound6.jnilib
  Reason: image not found
```

Reply:
Is there anything I can do to help get this merged?

The fact that I can’t reproduce the Java link error locally is weird. I suppose I could just remove the test of Java bindings; I don’t know of another formula that tests these.